Jetcaster/core/src/main/java/com/example/jetcaster/core/player/MockEpisodePlayer.kt

```diff
@@ -129,6 +133,35 @@ class MockEpisodePlayer(
         next()
     }
 
+    override fun play(playerEpisodes: List<PlayerEpisode>) {
+        if (isPlaying.value) {
```

Comment:
Maybe you could...

never mind...

I'm not 100% convinced this is the exact logic I'd expect in a player, but seems fine.